python/pyspark/pandas/sql_formatter.py

```diff
@@ -102,7 +102,7 @@ def sql(
             e      f       3  6
 
             Also note that the index name(s) should be matched to the existing name.
-    args : dict
+    args : dict or list
         A dictionary of parameter names to Python objects that can be converted to
```

Comment:
If args can be a list, should the description be somehow changed to include this option? It just says "A dictionary of parameters..."

Reply:
Yep, I am changing it now. Thank you.